### ISSUE TITLE
IsGeolocationEnabled Event

### ISSUE THREAD

**Issue by BullT** [jamesmontemagno/GeolocatorPlugin#258]
Hello,

I'd like implement a event to inform my when the IsGeolocationEnabled is enabled or not.

How can do it that?

Thanks,
XCO

**Comment by ericmclachlan** [comment]
I agree 100% with this suggestion. If the user disables the GPS on the device (e.g. boarding an aircraft) and later re-enables the GPS, the application is not able to resume picking up location changes.

Thanks for all the good work!
